PlatformRouting/PlatformRouting/_iOS/_App/MappedUIKitRouter.swift

```diff
@@ -341,6 +347,31 @@ open class MappedUIKitRouter: MappedRouter {
             completion?(nil, false)
         }
     }
+    
+    private func popup(_ viewController: UIViewController, animated: Bool, completion: RoutingCompletionBlock?) {
+        if let root = UIViewController.root() {
+            // wrapper is necessary for custom transitions
+            let wrapperViewController = UIViewController()
+            wrapperViewController.modalPresentationStyle = .overFullScreen // or .custom for other presentation styles
+            wrapperViewController.transitioningDelegate = fadeTransitioner
```

Comment:
tried this without a wrapper VC by directing transitioning to the UIHostingController, but that did not work. Wrapper VC is necessary for custom transitions